### ISSUE TITLE
Checkboxes and Tab selector buttons have no visible focus rendition

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#10625]
Tab could be focused here or one of the checkboxes. User cannot tell:

<img width="689" alt="Image" src="https://github.com/user-attachments/assets/9e8546cb-1f88-4401-a137-ff6f0e3a2fef" />
